### ISSUE TITLE
Logs: persist displayed fields in URL for sharing

### ISSUE THREAD

**Issue by matyax** [grafana/grafana#89168]
Some users use Show this field instead of the message in the logs panel, but are upset that when they share a link via the Copy shortlink the fields they selected to show are not saved, so they are forced to share screenshots instead to highlight the information they want to share.

**Comment by redbaron** [comment]
It seems that displayed fields state is not saved in the dashboard either, at least not when logs panel is in collapsable row

**Comment by matyax** [comment]
@redbaron Yes, that's why it's unchecked in the issue description. In the mean time, for everyone reading this, you can use the following trick: in your dashboard, go to Settings, then the JSON Model tab, look for your Logs panel, and add `displayedFields` to the options, like:

```
      "options": {
        "dedupStrategy": "none",
        "enableInfiniteScrolling": false,
        "enableLogDetails": true,
        "prettifyLogMessage": false,
        "showCommonLabels": false,
        "showLabels": false,
        "showTime": true,
        "sortOrder": "Descending",
        "wrapLogMessage": false
        // etc
        "displayedFields": ["fieldName1", "fieldName2"],
      },
```

For example: "displayedFields": ["time", "detected_level", "scope_name", "\_\_line\_\_"], where `__line__` is the actual log line, if you need it.

**Comment by matyax** [comment]
Explore and Drilldown work completed. Any dashboard related change will be done in #95553
